src/Mapping.php

```diff
@@ -179,12 +179,17 @@ public function extractValues($owner)
      * @param object $object
      * @return array
      */
-    private function extractObjectValues($object)
+    protected function extractObjectValues($object)
     {
-        $values = ArrayHelper::toArray($object, [], false);
+        if ($object instanceof yii\base\Model) { 
```

Comment:
This is obviously incorrect: either full qualified class name should be used (with leading '\' - `\yii\base\Model`) or `use` statement should be added.